packages/urban-bot/src/components/Dialog/DialogStep.tsx

```diff
@@ -3,7 +3,7 @@ import { matchPattern } from '../../utils/matchPattern';
 import { Text } from '../Text';
 import { useAction, useDialog, useText } from '../..';
 
-export type DialogValidationResult = string | void | Promise<string | void>;
+export type DialogValidationResult = ReactNode | Promise<ReactNode>;
```

Comment:
I suggest to left all current types and add ReactNode

Reply:
done